nbextensions/usability/ruler/main.js

```diff
@@ -26,28 +26,39 @@ define([
     };
 
     config.loaded.then(function() {
-        if (config.data.hasOwnProperty('ruler_color')) {
+console.log(config.data.ruler_color)
+console.log(config.data.ruler_color.length)
```

Comment:
Can you remove the `console.log` calls ?
The extension crashes if there is no `config.data.ruler_color`